src/desktop/wallpaper.rs

```diff
@@ -52,6 +52,7 @@ use crate::{
 
 #[derive(Serialize, Deserialize, Debug, Clone, Copy, PartialEq, Eq, Hash, Type)]
 #[zvariant(signature = "s")]
+#[serde(rename_all = "lowercase")]
```

Comment:
Is this needed only here ? might be worth checking other enums as well while you are at it.

Reply:
I checked all the pub enums and I think its only needed there.